setup.py

```diff
@@ -55,9 +55,10 @@ def get_readme() -> str:
         "stable-baselines~=2.10.0",
         "jax~=0.1.66",
         "jaxlib~=0.1.47",
-        # sacred==0.7.5 build is broken without pymongo
-        # sacred>0.7.4 have non-picklable config objects (see GH #109)
-        "sacred==0.7.4",
+        # TODO(shwang): Install from IDSIA/sacred@master once
+        # https://github.com/IDSIA/sacred/pull/737 and
+        # https://github.com/IDSIA/sacred/pull/740 are merged.
+        "sacred @ git+https://github.com/HumanCompatibleAI/sacred@0.8.1-with-imit-compat",  # noqa: E501
```

Comment:
We can't upload to PyPi if our requirements are not in PyPi, so I'm reluctant to make this change on a non-temporary basis.

Reply:
This is a reasonable objection, I forgot that PyPI has this requirement